app-api/src/main/java/org/openmrs/eip/app/config/ManagementDataSourceConfig.java

```diff
@@ -74,7 +75,7 @@ public PlatformTransactionManager transactionManager(@Qualifier("mngtEntityManag
 	
 	@Bean(name = "liquibase")
 	public SpringLiquibase getSpringLiquibaseForMgtDB(@Qualifier("mngtDataSource") DataSource dataSource, Environment env) {
-		SpringLiquibase liquibase = new SpringLiquibase();
+		BeanAwareSpringLiquibase liquibase = new BeanAwareSpringLiquibase();
```

Comment:
Why is this change necessary?

Reply:
Removed, as explained above. Now we are using SyncContext.